pkg/service/repl_test.go

```diff
@@ -31,11 +31,12 @@ type testReplSuite struct {
 func (s *testReplSuite) SetUpSuite(c *C) {
 	_, err := exec.LookPath("redis-server")
 	s.redisExists = (err == nil)
+	if !s.redisExists {
+		c.Skip("redis-server not found, skip replication tests")
+	}
```

Comment:
If no redis, we can still test qdb master <-> qdb slave replication, but here will skip all tests.